_ja/functions/aggregate.md

```diff
@@ -0,0 +1,12 @@
+---
+name: aggregate
+---
+
+# `aggregate`
+
+@include [signatures/aggregate.md]
+
+`aggregate` は二項演算子 `seqop` を要素のかたまりに適用し、二項演算子 `combop` を使って中間結果を結合します。
+値 `z` は型 `B` の<strong>零元</strong>であり、何回も使われるかもしれません。
```

Comment:
I guess that the `<strong>` tag comes because of the underscore for the "zero" element in the related docs, which becomes unparsed in japanese, right?

Indeed `_word_` is rendered as `<i>word</i>`. But you can use `*word*` as well to achieve the same effect.

It happens several times in other documents.

Reply:
A reason why HTML tags used is markdown parsing issue as you mentioned.

I intentionally used `<strong>` instead of `<i>`, since most Japanese fonts does not have italic style actually.
They have oblique style in which fonts are mechanically bended, not adjusted by font designers, so not eye-friendly. 
And, italic style is rarely used in Japanese texts to emphasize a phrase. 
Bold, underline or marker (background color) are more often used. 
I chosed bold `<strong>` (or could be `<b>`), since underline looks hypperlink, and marker requires css styling.

Reply:
It makes sense.

As a hint you also could use `**word**` syntax to render bold text.